tests/queries/0_stateless/00753_alter_attach.sql

```diff
@@ -90,3 +90,45 @@ OPTIMIZE TABLE partition_all2 PARTITION ALL; -- { serverError 344 }
 
 DROP TABLE partition_all;
 DROP TABLE partition_all2;
+
+-- test ATTACH ALL
+CREATE TABLE partition_attach_all (x UInt64, p UInt8) ENGINE = MergeTree ORDER BY x PARTITION BY p;
+INSERT INTO partition_attach_all VALUES (1, 1), (2, 2), (3, 3);
+
+ALTER TABLE partition_attach_all DETACH PARTITION ALL;
+SELECT * FROM partition_attach_all ORDER BY x;
+ALTER TABLE partition_attach_all ATTACH PARTITION ALL;
+SELECT * FROM partition_attach_all ORDER BY x;
+
+ALTER TABLE partition_attach_all DETACH PARTITION 1;
+SELECT * FROM partition_attach_all ORDER BY x;
+ALTER TABLE partition_attach_all ATTACH PARTITION ALL;
+SELECT * FROM partition_attach_all ORDER BY x;
+
+ALTER TABLE partition_attach_all DROP PARTITION ALL;
+SELECT * FROM partition_attach_all ORDER BY x;
+ALTER TABLE partition_attach_all ATTACH PARTITION ALL;
+SELECT * FROM partition_attach_all ORDER BY x;
+
+CREATE TABLE replicated_partition_attach_all (x UInt64, p UInt8)
+    ENGINE = ReplicatedMergeTree('/clickhouse/tables/test_00753_{database}/replicated_partition_attach_all', '1')
+        ORDER BY x
+        PARTITION BY p;
+
```

Comment:
Let's insert some data in the replicated table too.

Reply:
thanks, it's done. idk how i missed that